test/snippet/argument_parser/validators_output_directory.cpp

```diff
@@ -0,0 +1,31 @@
+//! [usage]
+#include <seqan3/argument_parser/all.hpp>
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/std/filesystem>
+
+int main(int argc, const char ** argv)
+{
+    seqan3::argument_parser myparser("Test", argc, argv); // initialize
+
+    //! [validator_call]
+    std::filesystem::path myfile;
+
+    myparser.add_option(myfile,'d',"dir","The output directory for storing the files.",
+                        seqan3::option_spec::DEFAULT, seqan3::output_directory_validator());
+    //! [validator_call]
+
+    // an exception will be thrown if the user specifies a directory name without trailing directory separator.
```

Comment:
you sure? I thought the is_directory checks for an extension?
also add a comment that is has to be writable

Reply:
No it needs to be adapted. Good catch